ChangeLog.md

```diff
@@ -1,4 +1,13 @@
-# Minicraft+ version 2.2.0
+# Minicraft+ Version 2 Changelog
+
+This changelog is related to all the Minicraft+ versions.
+All notable changes to this project will be documented in this file.
```

Comment:
This header is wrong.

Reply:
This changelog now only includes version 2 and later rather than after 1.7 as all logs from 1.7 to 1.9.4 are really mess to the markdown...

Reply:
> # Changelog
> 
> All notable changes to this project will be documented in this file.
> 
> The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
> and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

https://keepachangelog.com/en/1.0.0/

Reply:
I want to change a bit from the format.